src/Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem.tsx

```diff
@@ -34,36 +34,27 @@ export class ArtistAuctionResultItem extends Component<Props> {
         {({ user, mediator }) => {
           return (
             <Row>
-              <Responsive>
-                {({ xs, sm, md }) => {
-                  if (xs) {
-                    return (
-                      <ExtraSmallAuctionItem
-                        {...this.props}
-                        mediator={mediator}
-                        user={user}
-                      />
-                    )
-                  } else if (sm || md) {
-                    return (
-                      <SmallAuctionItem
-                        {...this.props}
-                        mediator={mediator}
-                        user={user}
-                      />
-                    )
-                  } else {
-                    return (
-                      <LargeAuctionItem
-                        {...this.props}
-                        mediator={mediator}
-                        user={user}
-                      />
-                    )
-                  }
-                }}
-              </Responsive>
-
+              <breakpoints.xs>
+                <ExtraSmallAuctionItem
+                  {...this.props}
+                  mediator={mediator}
+                  user={user}
+                />
+              </breakpoints.xs>
+              <Match any={[breakpoints.sm, breakpoints.md]}>
+                <SmallAuctionItem
+                  {...this.props}
+                  mediator={mediator}
+                  user={user}
+                />
+              </Match>
+              <Match any={[breakpoints.lg, breakpoints.xl]}>
+                <LargeAuctionItem
+                  {...this.props}
+                  mediator={mediator}
+                  user={user}
+                />
+              </Match>
```

Comment:
I'm inclined to think that rather than exporting `breakpoints` to be used directly, we create intermediary compositional components at the app (as opposed to lib) level: 

```jsx
export const breakpoints = {
  xs: (children) => <Match any={[breakpoints.xs]}>{children}</Match>
}
```

Reply:
Yep, good point. Addressed in Media2 👍